source/type/type.h

```diff
@@ -614,12 +614,14 @@ struct Line : public Header, Nameable, HasMessages, Codes{
     MultiLineString shape;
     boost::optional<boost::posix_time::time_duration> opening_time, closing_time;
 
+    std::map<std::string,std::string> properties;
```

Comment:
Why properties are only implemented for `Line` even if, in ed, it can be route, stop_area or stop_point?